### ISSUE TITLE
greater than (e.g. >=3.6) not supported

### ISSUE THREAD

**Issue by tacman** [phar-io/version#34]
I'm trying to parse the version string 

     symfony/monolog-bundle: >=3.6 

on https://packagist.org/packages/antonchernik/logging-bundle,

```
3^ PharIo\Version\UnsupportedVersionConstraintException^ {#4853
  #message: "Version constraint >=3.6 is not supported."
  #code: 0
  #file: "./vendor/phar-io/version/src/VersionConstraintParser.php"
  #line: 22
  trace: {
    ./vendor/phar-io/version/src/VersionConstraintParser.php:22 { …}
```

**Comment by theseer** [comment]
We so far did not implement these as their use is discouraged.
Using unbounded constraints like these is a disaster waiting to happen but I can see that we might have to support (read: at least understand) it.

**Comment by tacman** [comment]
Please consider it, I'm working on an open-source app that allows searching packagist for Symfony bundles that meet certain criteria (like at least php 8.2 and Symfony at least 6.4 or 7.0).  I can assure you that LOTS of packages use this format, and even the composer docs talk about it. 

> When you write a version constraint, it may reference a specific tag (e.g., 1.1) or it may reference a valid range of tags (e.g., >=1.1 <2.0, or ~4.0). 

https://getcomposer.org/doc/articles/versions.md#tags

**Comment by tacman** [comment]
I agree with you on the problems.  In fact, many bundles are returned as being compatible with Symfony 7 because the constrait is ">= 2.2".  Terrible.

BUT many of the php constraints do work, even php >= 7.2 often works, where php ^7.2 fails with PHP 8, even if php 8 works. And getting authors of these old libraries to accept PRs is often difficult.
